### ISSUE TITLE
Use ComparableX509 and ComparableRSAKey

### ISSUE THREAD

**Issue by kuba** [kuba/simp_le#40]
For example, for data imported from the official client:

```
# simp_le -d ${domain?}:/var/www/html --account_key_size 2048 -f account_key.json  -f key.pem -f cert.pem -f fullchain.pem -f chain.pem
Some plugins returned conflicting data for the "chain" component
Debugging tips: -v improves output verbosity. Help is available under --help.
```

This is because `OpenSSL.crypto.X509.__eq__` is not defined sensibly. Similar applies to `-f key.pem -f key.cert` et al (`OpenSSL.crypto.PKey.__eq__` broken).

`acme` library has convenience wrappers: `jose.ComparableX509` and `jose.ComparableRSAKey` that should be used throughout `simp_le` code

**Comment by kuba** [comment]
1. We're already using `ComparableX509`. The problem related to data imported from the official client is actually caused by `simp_le` requiring `\n\n` to be used to separate 2 certs. Quickfix: `perl -0777 -pie 's/-\n-/-\n\n-/g' fullchain.pem`
2. `ComparableRSAKey` is a wrapper for `cryptography` keys. It seems that new class needs to be created... WIP

**Comment by kuba** [comment]
(2) is fixed in 6e895dbb47a8fe27276ec846c0ce7061fb49eb42

**Comment by kuba** [comment]
For (1) I've opened #42. 6e895db is now in master
